cli/src/commands/extensions/api.rs

```diff
@@ -393,6 +397,30 @@ async fn parse_lockfile(
     Ok(PackageLock { packages: parsed.packages, format: parsed.format })
 }
 
+/// Return error when trying to sandbox on Windows.
+#[op]
+#[cfg(not(unix))]
+fn run_sandboxed(process: Process) -> Result<ProcessOutput> {
+    if !process.exceptions.best_effort {
+        Err(anyhow!("Extension sandboxing is not supported on this platform"))
+    }
+
```

Comment:
We've got a debug message when the min landlock version isn't met. We should probably have a similar message here (and anywhere else that we use `best_effort` in the future).

```suggestion
    log::debug!("Sandboxing unsupported on this platform, skipping sandbox")

```